qiskit/circuit/library/basis_change/qft.py

```diff
@@ -130,10 +130,9 @@ def num_qubits(self, num_qubits: int) -> None:
         if num_qubits != self.num_qubits:
             self._invalidate()
 
-            if num_qubits:
+            self.qregs: List[QuantumRegister] = []
```

Comment:
From https://dev.azure.com/qiskit-ci/qiskit-terra/_build/results?buildId=31187&view=logs&jobId=f0ab2c38-58fa-55ad-8e8b-d3984171f737&j=f0ab2c38-58fa-55ad-8e8b-d3984171f737&t=980625fc-4a64-50e4-7ca0-679c2a4bbda8, it looks like there's a lint failure here from not importing `List`.

Reply:
Since Elena finished her internship and is probably taking some Qiskit-free time I went ahead and fixed it 🙂